src/database/mountainsData.js

```diff
@@ -0,0 +1,108 @@
+const mountainsData = [
+  {
+    name: 'Greater Chimgan',
+    latitude: 41.4951900,
+    longitude: 70.0586718,
+    elevation: 3309.6,
+    image: 'mnt/mnt_01.svg',
+    current_weather: {
+      temperature: null,
+      windspeed: null,
+      windderection: null,
+      time: null,
+    },
+  },
+  {
+    name: 'Karakush',
+    latitude: 41.2477890,
+    longitude: 70.3391467,
+    elevation: 3864.3,
+    image: 'mnt/mnt_02.svg',
+    current_weather: {
+      temperature: null,
+      windspeed: null,
+      windderection: null,
+      time: null,
+    },
+  },
+  {
+    name: 'Babaitag',
+    latitude: 41.1549650,
+    longitude: 70.2232660,
+    elevation: 3555.7,
+    image: 'mnt/mnt_03.svg',
+    current_weather: {
+      temperature: null,
+      windspeed: null,
+      windderection: null,
+      time: null,
+    },
+  },
+  {
+    name: 'Kizilbash',
+    latitude: 42.0727687,
+    longitude: 70.4667018,
+    elevation: 4200,
+    image: 'mnt/mnt_04.svg',
+    current_weather: {
+      temperature: null,
+      windspeed: null,
+      windderection: null,
+      time: null,
+    },
+  },
+  {
+    name: 'Tavalgan',
+    latitude: 41.7915524,
+    longitude: 70.4090762,
+    elevation: 3888.8,
+    image: 'mnt/mnt_05.svg',
+    current_weather: {
+      temperature: null,
+      windspeed: null,
+      windderection: null,
+      time: null,
+    },
+  },
+  {
+    name: 'Adelunga',
+    latitude: 42.1235559,
+    longitude: 70.9689168,
+    elevation: 4301.8,
+    image: 'mnt/mnt_06.svg',
+    current_weather: {
+      temperature: null,
+      windspeed: null,
+      windderection: null,
+      time: null,
+    },
+  },
+  {
+    name: 'Xodja Gur Gur Ota',
+    latitude: 38.3828229,
+    longitude: 67.2788411,
+    elevation: 3723.7,
+    image: 'mnt/mnt_07.svg',
+    current_weather: {
+      temperature: null,
+      windspeed: null,
+      windderection: null,
+      time: null,
+    },
+  },
+  {
+    name: 'Hazret Sultan',
+    latitude: 38.9477782,
+    longitude: 68.1724695,
+    elevation: 4643,
+    image: 'mnt/mnt_08.svg',
+    current_weather: {
+      temperature: null,
+      windspeed: null,
+      windderection: null,
+      time: null,
+    },
+  },
+];
```

Comment:
- [ ] Hey Zilola, I appreciate the effort in creating a local database here but part of the project requirements is to use an API that provides numeric values based on a parameter and as such I'd advise you explore APIs that fit the app you already created so you can reduce rework.
- You can check out [this link](https://github.com/public-apis/public-apis) for public APIs.
- Here's the project requirement as a reminder:
  ![image](https://github.com/Zilola-Nazarova/mountains-forecast-uz/assets/104968236/3cc5e8ea-b174-44c8-8ab9-6a43dfd87767)

Reply:
Hi @AbelG101 
I explained in the video that we don't have such a database
And I think I followed the project requirements because I retrieved data about weather conditions from API and displayed it in my App based on the mountain coordinates

Reply:
Please watch my video and tell me what you think about the icons that are shown for each mountain card.
Please take into account that these icons don't come from my local database. I retrieve information about the weather_code from API but instead of showing a number I thought it would be more interactive to show an icon depending on that weather_code